username_api.py

```diff
@@ -184,9 +183,9 @@ def check_usable(website):
 
 
 def check_format(website, username):
-    """Check the format of a username depending on the website
     """
-
+    Check the format of a username depending on the website.
```

Comment:
this one is a different style than the others :/

This style is the style which coala can support, not the `"""Do foo.\n..."""` style